Justfile

```diff
@@ -366,7 +377,9 @@ function terra_caseify()
       local download_conda=0
       local conda_install
 
-      : ${PYTHON_VERSION=3.7.13}
+      : ${PYTHON_VERSION=3.8.16}
+      : ${PIPENV_VERSION=2023.4.29}
+      : ${VIRTUALENV_VERSION=20.23.0}
```

Comment:
Can we allow override via some `TERRA_*` variables? 
```
: ${PYTHON_VERSION=${TERRA_PYTHON_VERSION:-3.8.16}}
: ${PIPENV_VERSION=${TERRA_PIPENV_VERSION:-2023.4.29}}
: ${VIRTUALENV_VERSION=${TERRA_VIRTUALENV_VERSION:-20.23.0}}
```

Reply:
```suggestion
      : ${PYTHON_VERSION=${TERRA_PYTHON_VERSION:-3.8.16}}
      : ${PIPENV_VERSION=${TERRA_PIPENV_VERSION:-2023.4.29}}
      : ${VIRTUALENV_VERSION=${TERRA_VIRTUALENV_VERSION:-20.23.0}}
```